blocks/rich-text/index.js

```diff
@@ -837,6 +928,26 @@ export class RichText extends Component {
 						{ formatToolbar }
 					</div>
 				) }
+				{ isSelected && isInlineInsertionPointVisible &&
+					<InlineInsertionPoint
```

Comment:
I have no idea here from a quick look, but what prevents us from positioning the parent element above, and keeping a reference to that, instead of calculating the positioned parent element?

Reply:
Good question! I had assumed since there was existing code that calculated the positioned parent, that it must be necessary. But looking at it now I can't find a reason for it. Commit https://github.com/WordPress/gutenberg/pull/5794/commits/4abeaf72c7eec9d5450969273070323771196b28 uses a `ref` instead, as you suggested.

I wonder if `getFocusPosition` can use the `ref` as well? Since it is existing code that is used to position the `FormatToolbar`, I will refrain from changing it there for now, in case there is a reason for it.